### ISSUE TITLE
Miri CI invocation needs updating

### ISSUE THREAD

**Issue by RalfJung** [jonhoo/flurry#93]
`cargo miri test` changed to be more compatible with `cargo test`, so the old style ([used by this repo](https://github.com/jonhoo/flurry/blob/9a443cbd393c1605eb6df302c0d7531d70eb7be6/azure-pipelines.yml#L60)) of running `cargo miri test -- -Zmiri-ignore-leaks` is deprecated. The new way is `MIRIFLAGS="-Zmiri-ignore-leaks" cargo miri test`.

I'm not familiar with Azure CI or else I'd have prepared a PR. I hope this is not too much of an effort. :)

**Comment by RalfJung** [comment]
I also found [another project of yours which needs a similar update](https://github.com/jonhoo/griddle/blob/4e03134b254e5ed72b3068f9c70e7ce2d55b42c0/azure-pipelines.yml#L48).

**Comment by jonhoo** [comment]
Ah, good catch! Should be easy enough to fix. Just need to replace
```yaml
     - script: cargo miri test -- -Zmiri-ignore-leaks
       displayName: cargo miri test
```
with
```yaml
     - script: cargo miri test
       displayName: cargo miri test
       env:
         MIRIFLAGS: -Zmiri-ignore-leaks
```
I'm pretty swamped these days with thesis writing, but will keep the issue open so I (hopefully) remember when my schedule settles down a little :sweat_smile:

**Comment by jonhoo** [comment]
Closed by #94. Thanks @RalfJung!
